src/Features/LanguageServer/Protocol/Extensions/ProtocolConversions.cs

```diff
@@ -194,9 +194,12 @@ public static LSP.VersionedTextDocumentIdentifier DocumentToVersionedTextDocumen
         public static LinePosition PositionToLinePosition(LSP.Position position)
             => new LinePosition(position.Line, position.Character);
 
+        public static LinePositionSpan RangeToLinePositionSpan(LSP.Range range)
+            => new(PositionToLinePosition(range.Start), PositionToLinePosition(range.End));
```

Comment:
📝 This is a restoration of a helper method that was recently removed in #68081 due to lack of use